.github/scripts/mocks/azure-mock.yaml

```
@@ -46,7 +53,20 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: azure-mock
+  annotations:
+    cert-manager.io/cluster-issuer: "artesca-root-ca-issuer"
 spec:
+  tls:
+    - secretName: zenko-tls-azure
+      hosts:
+        - "*.azure-mock.zenko.local"
+        - "azure-mock.zenko.local"
+        - "*.blob.azure-mock.zenko.local"
+        - "blob.azure-mock.zenko.local"
+        - "*.queue.azure-mock.zenko.local"
+        - "queue.azure-mock.zenko.local"
+        - "devstoreaccount1.blob.azure-mock.zenko.local"
+        - "devstoreaccount1.queue.azure-mock.zenko.local"
```

many not needed, should be covered by the wildcards above already

```suggestion
        - "azure-mock.zenko.local"
        - "*.azure-mock.zenko.local"
        - "*.blob.azure-mock.zenko.local"
        - "*.queue.azure-mock.zenko.local"
```